public/data/vedanshi.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Vedanshi Prajapati",
+  "type": "personal",
+  "bio": "I am vedanshi, I’m currently learning Data Structures & Algorithms, I'm open to contribute into new open-source projects, Pronouns: she/her. ",
+  "avatar": "https://github.com/Tokio3012.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Tokio3012",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/tokio3012",
+      "icon": "twitter"
+    },
+    {
+      "name": "connect with me in discord",
+      "url": "https://discordapp.com/tokio/7761/",
```

Comment:
There is a broken link, please update it with the correct one! 🛠

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant